scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -131,42 +137,106 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
      */
     fun getDetails() = ScannerDetails(scannerName, getVersion(), getConfiguration())
 
-    override fun scanPackages(packages: List<Package>, outputDirectory: File, downloadDirectory: File):
+    // `newFixedThreadPoolContext` is obsolete and will be replaced with a new API, see:
```

Comment:
Instead of saying that the API is obsolete (which is obvious from the annotation), I'd prefer this comment to say *why* we still use the obsolete API.

Reply:
Done.